packages/cozy-procedures/src/components/overview/helpers.js

```diff
@@ -0,0 +1,86 @@
+import formatDate from 'date-fns/format'
+import subMonths from 'date-fns/sub_months'
+import startOfMonth from 'date-fns/start_of_month'
+import pick from 'lodash/pick'
+
+/**
+ * createJsonFile - upload a json file containing the given data
+ *
+ * @param {string} filename - The json file name
+ * @param {string} destinationId - The id of the destination directory
+ * @param {string} content - The file content
+ * @return {object} Information about the jsonFile (id and name)
+ */
+export const createJsonFile = async (
+  client,
+  filename,
+  destinationId,
+  content
+) => {
+  const file = new File([content], filename)
+  const resp = await client
+    .collection('io.cozy.files')
+    .createFile(file, { dirId: destinationId })
+
+  return resp.data
+}
+
+export const fetchCheckingsAccounts = async client => {
+  const query = client.find('io.cozy.bank.accounts').where({
+    type: {
+      $in: ['Checkings', 'Bank', 'Cash', 'Deposit']
+    }
+  })
+
+  // TODO extract this loop to cozy-client
```

Comment:
The queryAll is in the stack-client and not in cozy-client ?

Reply:
See https://github.com/cozy/cozy-client/pull/486